### DISCUSSION TITLE
I want to configure a static service with nuxt3

### DISCUSSION THREAD

**Discussion by SnowGuest** [nuxt/nuxt#17178] [Questions]
I have a remote static image resource service, this service cannot be accessed directly in the browser, there is always a problem, so I want to use the nuxt3 proxy to request this static image service, how can I configure it to proxy it

**Comment by Flanker72** [comment]
You may create some [api](https://v3.nuxtjs.org/guide/features/server-routes/) to work with such remote service.
I assume, your API should provide some get and post handlers to work with images (to fetch and upload them). And inside this API you may communicate with your remote service via some server-side fetch/stream APIs.
Then from the frontend part you should request your `/api/<HANDLER>` to work with this service.
